src/webviews/src/App.tsx

```diff
@@ -73,52 +78,59 @@ function App({}: AppProps) {
 
   return (
     <div className="p-4">
-      <div className="mb-2 p-4">
-        <h1 className="text-2xl font-bold text-vscode-settings-headerForeground py-2">
-          Flat Editor
-        </h1>
-        <p className="mb-2">
-          This is a gui for setting up a Flat Action, which will pull external
-          data and update it using GitHub Actions.
-        </p>
-        <button onClick={handleOpenRaw}>View the raw YAML</button>
-      </div>
-      <Triggers />
-      <Jobs />
-
-      {showErrorState ? (
-        <div className="border-2 border-vscode-editorOverviewRuler-warningForeground bg-vscode-settings-dropdownBackground text-sm p-4 flex items-center">
-          <span className="codicon codicon-warning mr-1 text-vscode-editorOverviewRuler-warningForeground" />
-          <p>
-            Make sure all of your steps have a{' '}
-            <strong>downloaded_filename</strong> specified!
-          </p>
+      <header>
+        <div className="mb-2">
+          <h1 className="text-[26px] leading-[30px] font-medium my-0">
+            Flat Editor
+          </h1>
         </div>
-      ) : (
-        <div className="pt-4 pl-4 pb-6">
-          <p className="text-lg font-bold pb-1">
-            You're all set!{' '}
-            <span
-              className="codicon codicon-rocket pl-1"
-              style={{ fontSize: '0.875rem' }}
-            />
-          </p>
-          <p>
-            Commit, push, and check out your new Action{' '}
-            {actionsUrl ? (
-              <span>
-                <a className="underline" href={actionsUrl}>
-                  on GitHub
-                </a>
-                .
-              </span>
-            ) : (
-              'on GitHub.  '
-            )}{' '}
-            It should run automatically, once pushed.
+        <div className="mb-4">
+          <p className="text-[13px] font-medium my-0">
+            This is a gui for setting up a Flat Action, which will pull external
+            data and update it using GitHub Actions.
           </p>
         </div>
-      )}
+        <VSCodeButton onClick={handleOpenRaw}>View the raw YAML</VSCodeButton>
+      </header>
+      <div className="my-4">
+        <VSCodeDivider />
+      </div>
+      <div className="space-y-6">
+        <Triggers />
+        <Jobs />
+      </div>
+      <div className="my-4">
+        <VSCodeDivider />
+      </div>
+      <div>
+        {showErrorState ? (
+          <div className="text-[color:var(--vscode-errorForeground)] flex items-center">
+            <span className="codicon codicon-warning" />
+            <p className="ml-1">
+              Make sure all of your steps have a{' '}
+              <strong>downloaded_filename</strong> specified!
```

Comment:
Just wanted to double-check, is it intended that "downloaded_filename" will be semantically emphasized (i.e. screen readers will distinguish this text as important) or is the usage of `strong` just for bolding the text?

Reply:
If it's just for bolding text, you should switch out the strong tag with a span tag and use CSS to change the font-weight